bitnami/postgresql/README.md

```diff
@@ -2,12 +2,14 @@
 
 # PostgreSQL packaged by Bitnami
 
-PostgreSQL (Postgres) is an open source object-relational database known for reliability and data integrity. ACID-compliant, it supports foreign keys, joins, views, triggers and stored procedures.
+PostgreSQL (Postgres) is an open source object-relational database known for reliability and data integrity.
```

Comment:
Is there any reason to break the lines? Please follow the same conventions we already have. Could you also let bitnami-bot run the readme generator in your repository? We are not able to merge any external PR with [failures in the pipeline](https://github.com/bitnami/charts/runs/6550236832?check_suite_focus=true)

Reply:
My IDE did that, I will revert that change.